python_on_whales/components/compose/cli_wrapper.py

```diff
@@ -663,7 +663,7 @@ def run(
         command: List[str] = [],
         build: bool = False,
         detach: bool = False,
-        # entrypoint: Optional[List[str]] = None,
+        entrypoint: Optional[List[str]] = None,
```

Comment:
```suggestion
        entrypoint: Optional[str] = None,
```
In the tests you're passing a string, so I believe that should be wrong to expect a `List[str]`